adafruit_turtle.py

```diff
@@ -1,7 +1,6 @@
 # SPDX-FileCopyrightText: 2006-2010 Gregor Lingl for Adafruit Industries
 # SPDX-FileCopyrightText: 2019 LadyAda for Adafruit Industries
 # SPDX-FileCopyrightText: 2021 Dave Astels for Adafruit Industries
-# SPDX-FileCopyrightText: 2022 Carsten Thue-Bludworth for Adafruit Industries
```

Comment:
If you'd like to keep yourself credited, you can definitely do that, just without the `for Adafruit Industries` portion. It is up to you though.